### DISCUSSION TITLE
Open the example network activation based fem_ approve.srn5, the execution program has an error prompt.

### DISCUSSION THREAD

**Discussion by zhifeiji-han** [SCIInstitute/SCIRun#2284] [General] [answered]
Open the example network activation based fem_ approve.srn5, the execution program has an error prompt. The calculatedistancedofield module prompts the error tip "error: input data required on port inputfield".
![image](https://user-images.githubusercontent.com/70640243/116199107-55557280-a769-11eb-95e1-dc06df24dd13.png)

**Comment by jessdtate** [comment, marked as answer]
It looks like one of the connections has been deleted, and I think it is due to a missing module.  I believe it is supposed to be interfacewithtetgen.  You can get that module by building from source with Tetgen  enabled.   you would have to tune the meshing parameters too.  

Also, please make sure that you read the module note in the top module.  It may not be worth getting it working for you.

**Comment by jessdtate** [comment]
Also this bug report is in the FwdInvToolkit.

**Comment by zhifeiji-han** [comment]
I see. Thank you!
